src/vt/configs/arguments/args.cc

```diff
@@ -52,12 +53,170 @@
 #include <ctime>
 
 #include "CLI/CLI11.hpp"
+#include "yaml-cpp/yaml.h"
 
 namespace vt { namespace arguments {
 
 // Temporary variables used only for parsing artifacts.
 namespace {
 
+// Set the YAML labels for all CLI args
+
+// Output Control
+static const std::string vt_color_label = "Color";
+static const std::string vt_no_color_label = "Color";
```

Comment:
This should probably be:
```suggestion
static const std::string vt_no_color_label = "No color";
```

Reply:
Fixed